src/contracts/core/AllocationManager.sol

```diff
@@ -905,4 +1008,45 @@ contract AllocationManager is
         // less than or equal to
         return status.registered || block.number <= status.slashableUntil;
     }
+
+    /// @inheritdoc IAllocationManager
+    function getRedistributionRecipient(
+        OperatorSet memory operatorSet
+    ) public view returns (address) {
+        // Load the redistribution recipient and return it if set, otherwise return the default burn address.
+        address redistributionRecipient = _redistributionRecipients[operatorSet.key()];
+        return redistributionRecipient == address(0) ? DEFAULT_BURN_ADDRESS : redistributionRecipient;
+    }
+
+    /// @inheritdoc IAllocationManager
+    function isRedistributingOperatorSet(
+        OperatorSet memory operatorSet
+    ) public view returns (bool) {
+        return _redistributionRecipients[operatorSet.key()] != address(0);
+    }
+
+    /// @inheritdoc IAllocationManager
+    function getSlashCount(
+        OperatorSet memory operatorSet
+    ) external view returns (uint256) {
+        return _slashCount[operatorSet.key()];
+    }
+
+    /// @inheritdoc IAllocationManager
+    function isOperatorRedistributable(
+        address operator
+    ) external view returns (bool) {
+        EnumerableSet.Bytes32Set storage allocatedSets = allocatedSets[operator];
+        EnumerableSet.Bytes32Set storage registeredSets = registeredSets[operator];
```

Comment:
You are shadowing storage variables here. If you want to use storage pointers, please rename them to avoid shadowing. Prefacing with an underscore is fine.